core/src/main/java/com/datastax/oss/driver/internal/core/metadata/schema/DefaultAggregateMetadata.java

```diff
@@ -109,6 +110,13 @@ public DataType getStateType() {
     if (initCond == null) {
       return Optional.empty();
     }
+    if (stateTypeCodec == null) {
+      LOG.debug(
+          String.format(
+              "Null codec for INITCOND for %s.%s, using toString instead.  This often indicates use of a deserialized AggregateMetadata instance.",
+              keyspace.asInternal(), signature.getName().asInternal()));
+      return Optional.of(initCond.toString());
+    }
```

Comment:
This is not strictly necessary; the try/catch block below will catch the NPE in cases where stateTypeCodec is null from deserialization.  I added the explicit case here only to avoid adding messages at WARN to the logging infrastructure... DEBUG seemed like a more reasonable level for this (admittedly probably rare in practice) case.

Reply:
Yeah, I think I agree.  I mean the describe() op does have a fallback to toString() if the codec chucks an exception while doing it's formatting... but I agree it's probably better to avoid the transient/nullable stuff if we can.

I'll make the change.